tests/D2L.CodeStyle.TestAnalyzers.Tests/NUnit/ConfigTestSetupStringsAnalyzerTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using D2L.CodeStyle.TestAnalyzers.Common;

```

Comment:
Tests like this are fine, but our prefered way to write tests these days is.....
nevermind, not applicable to test analyzers. arghhhhhhhhhhhhhhhhhhh.